### PR TITLE
Remove |ignoreMutedMedia| and references to it. (#99)

### PR DESCRIPTION
**Pull request by yell0wd0g**

### CODE REVIEW

File: MediaRecorder.html [hunk 1]
```diff
@@ -326,12 +313,6 @@
             playable, but the combination of all the Blobs from a completed
             recording MUST be playable.</p>
 
-            <p> If any Track within the MediaStream is muted at any time (i.e.,
```
Comment (alvestrand) [inline MediaRecorder.html:329]: I think you should keep the "record black and silence" text, just to be explicit.

**Review by alvestrand — APPROVED** [review]
Good resolution. Suggest leaving in "record black and silence" for clarity.

**Review by jan-ivar — APPROVED** [review]
Lgtm, but I agree with @alvestrand. Also I think we meant `muted` or disabled  (`!enabled`) here FWIW (unsure how to write that).

File: MediaRecorder.html (hunk 1 above)
Comment (yell0wd0g) [reply MediaRecorder.html:329]: Rewritten

**Comment by yell0wd0g** [comment]
I restated the previously removed paragraph saying that only black frames and/or silence will be recorded, since that's what a Track should produce when `muted` or `disabled` (https://www.w3.org/TR/mediacapture-streams/#track-enabled) : 
> the consumer gets zero-information-content, which means silence for audio and black frames for video.

Although I believe that's not what Chromium implementation does: instead of black frames or silence, data sinks will get nothing at all if the track is muted or disabled.  But this pertains to implementation bugs.

**Pull request merged**